src/components/notifications/TechulusPush.vue

```diff
@@ -4,6 +4,53 @@
         <HiddenInput id="push-api-key" v-model="$parent.notification.pushAPIKey" :required="true" autocomplete="new-password"></HiddenInput>
     </div>
 
+    <div class="mb-3">
+        <label for="push-api-title" class="form-label">{{ $t("Title") }}</label>
+        <input id="push-api-title" v-model="$parent.notification.pushTitle" type="text" class="form-control">
+    </div>
+
+    <div class="mb-3">
+        <label for="push-api-channel" class="form-label">{{ $t("Channel") }}</label>
+        <input id="push-api-channel" v-model="$parent.notification.pushChannel" type="text" class="form-control">
```

Comment:
lets add a `pattern` here to make the helptext enforcement simpler

```suggestion
        <input id="push-api-channel" v-model="$parent.notification.pushChannel" type="text" class="form-control" patttern="[A-Za-z0-9-]+">
```